src/codegen/sys/lib_.rs

```diff
@@ -104,6 +105,21 @@ fn generate_lib(w: &mut dyn Write, env: &Env) -> Result<()> {
     Ok(())
 }
 
+fn generate_extern_crates(w: &mut dyn Write, env: &Env) -> Result<()> {
+    for library in &env.config.external_libraries {
+        w.write_all(
+            format!(
+                "use {}_sys as {};\n",
```

Comment:
Why not just use `foo_sys` everywhere directly instead of going via the rename?

Reply:
I tried to do that, but the whole code is a mess. So for now, I just want a single file to update whenever I do a release

Reply:
Ok, so this is just a hack for now to make things work and at a later time the whole renaming should be removed?

Maybe something @GuillaumeGomez wants to look at?

Reply:
> Ok, so this is just a hack for now to make things work and at a later time the whole renaming should be removed?

Yeah, just going back to the original thing we had